lib/lobstersbot/plugins/title.rb

```diff
@@ -0,0 +1,76 @@
+require 'net/http'
+
+require 'lobstersbot/plugins/title/wikipedia'
+require 'lobstersbot/plugins/title/lobsters'
+
+StopIteration = Class.new(StandardError)
+
+def parse_title(response)
```

Comment:
Can we move this into a module and use `throw/catch` if we don't need the stack trace?